File: day3/Quiz2/solution.py

def umPrimorial(number):
    result = 1
    count1 = 0
    for j in range(2,100) :
        count2 = 0
        for x in range(2,j+1):
            if j%x == 0:
                count2 += 1
            if count2 >=2 :
                break
        
        if count2 == 1:
            result = result*j
            count1 += 1
        if count1 == number:
            break
        
    print(result)
    

if __name__ == "__main__":
    umPrimorial(3)

File: day3/Quiz2/solution.java

public class MyClass {
    public static int umPrimorial(int number){
        int result = 1 ;
        int cnt1 = 0;
        for(int i = 2; i <= 100; i++  ){
            int cnt2 = 0;
            
            for(int j = 2; j <= i; j++){
                if(i%j==0){
                    cnt2++;
                }
                if(cnt2 >= 2){
                    break;
                }
            }
            if(cnt2 == 1 ){
                result = result* i;
                cnt1++;
            }
            if(cnt1 == number){
                break;
            }
            
        }

        return result;
        
    }
    public static void main(String args[]) {
        
        System.out.println(umPrimorial (3));
    }
}
